docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -908,6 +909,16 @@ You can also disable page caching for an individual [fixture](test-api/fixture/d
 
 *Related configuration file property*: [disablePageCaching](configuration-file.md#disablepagecaching).
 
+### --disable-multiple-windows
+
+Disables support for multi-window testing.
+
+```sh
+testcafe firefox my-tests --disable-multiple-windows
+```
+
+The `--disable-multiple-windows` option disables multi-window testing capabilities in Chrome and Firefox. Use it if the introduction of this feature breaks your legacy tests.
```

Comment:
> Use it if the introduction of this feature breaks your legacy tests.

We can say it in a milder manner, like:

`Use this flag if you encounter compatibility issues with [TestCafe v2015.1](http://testcafe.devexpress.com/documentation) tests.`